bftengine/src/bftengine/RequestHandler.cpp

```diff
@@ -149,7 +150,14 @@ void RequestHandler::execute(IRequestsHandler::ExecutionRequestsQueue& requests,
       req.outActualReplySize = 1;
     }
   }
-  if (userRequestsHandler_) return userRequestsHandler_->execute(requests, timestamp, batchCid, parent_span);
+  if (userRequestsHandler_) {
+    auto isPost = (requests.size() > 0 && !(requests.back().flags & bftEngine::PRE_PROCESS_FLAG));
+    ISystemResourceEntity::scopedDurMeasurment m(
```

Comment:
This is not accurate.
(1) you don't include here reconfiguration, key exchange, and ticks.
(2) you do include here read requests which are not going through a consensus (you need to filter them out) see https://github.com/vmware/concord-bft/blob/master/bftengine/src/bftengine/ReplicaImp.cpp#L4638

Reply:
regarding 1 - I think that the internal requests are neglectable relative to the application requests, though I can make it more general.
Regarding 2- good point, I'll filter them.

Reply:
regarding the calculations - they are being triggered by the getMeasurement method, which will be called from another thread than the main thread. 
I'll add a flag on the addMeasurement in order to control measurement addition for performance evaluation.

Reply:
Thanks